pkgs/applications/misc/holochain-launcher-bin/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, pkg-config
+, dpkg
+, openssl
+, webkitgtk
+, libappindicator
+, wrapGAppsHook
+}:
+
+let
+  version = "0.5.1";
+in
+
+stdenv.mkDerivation {
+  name = "holochain_launcher_bin";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://github.com/holochain/launcher/releases/download/v${version}/holochain-launcher_${version}_amd64.deb";
+    sha256 = "sha256-SJzDQQ3ltieZEoLsqEGnXskQcc+0n0Dwu36w59G87Qg=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    pkg-config
+    dpkg
+    # required for FileChooser
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    openssl
+    webkitgtk
+    libappindicator
+  ];
+
+  preFixup = ''
+    patchelf --add-needed "libappindicator3.so" "$out/bin/holochain-launcher"
+  '';
+
+  unpackCmd = "dpkg-deb -x $curSrc source";
+
+  installPhase = "mv usr $out";
```

Comment:
It is more of a recommendation, but could you please set this in the following order:
- unpackCmd
- installPhase
- preFixup